package.json

```diff
@@ -99,6 +99,7 @@
     "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
     "browserslist": "^4.21.3",
     "circular-dependency-plugin": "^5.2.2",
+    "chalk": "*",
```

Comment:
`chalk` is already used by other packages, so we need any version that already exists.